packages/client/src/components/ui/icons/pokt-icon.tsx

```
@@ -0,0 +1,28 @@
+import { LucideIcon } from "lucide-react"
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)